v21.1/community-tooling.md

```diff
@@ -20,6 +20,7 @@ If you have a tested or developed a third-party tool with CockroachDB, and would
 - [php-pgsql](build-a-php-app-with-cockroachdb.html)
 - [RPostgres](https://blog.ervits.com/2020/02/exploring-cockroachdb-with-r-and.html)
 - [Rust-Postgres](build-a-rust-app-with-cockroachdb.html)
+- [SchemaHero](https://schemahero.io/docs/installing/kubectl/)
```

Comment:
@laverya Can we link to the page mentioning CockroachDB support, https://schemahero.io/databases/cockroachdb/connecting/?